libcontainer/integration/execin_test.go

```diff
@@ -133,16 +133,23 @@ func TestExecInError(t *testing.T) {
 	}()
 	ok(t, err)
 
-	unexistent := &libcontainer.Process{
-		Args: []string{"unexistent"},
-		Env:  standardEnvironment,
-	}
-	err = container.Start(unexistent)
-	if err == nil {
-		t.Fatal("Should be an error")
-	}
-	if !strings.Contains(err.Error(), "executable file not found") {
-		t.Fatalf("Should be error about not found executable, got %s", err)
+	for i := 0; i < 42; i++ {
```

Comment:
We wouldn't want to loop until failure, though, otherwise the test will fail when it is working perfectly.
Yes 42 is a magic number.